src/olympia/addons/views.py

```diff
@@ -760,6 +761,17 @@ def check_object_permissions(self, request, obj):
     def get_queryset(self):
         """Return the right base queryset depending on the situation."""
         requested = self.request.GET.get('filter')
+        valid_filters = ('all_with_deleted',
+                         'all_with_unlisted',
+                         'all_without_unlisted',
+                         'only_beta')
```

Comment:
We are using something like

```
valid_filters = (
    'all_with_deleted',
    '...')
```

most of the time these days and I feel like it looks better (I'm actually a fan of putting the closing paren on the next line too)